test/test_helper.rb

```diff
@@ -1,6 +1,9 @@
+gem "minitest"
 require 'minitest/autorun'
 require 'minitest/pride' # Color!
 require 'mocha/mini_test'
+require 'webmock/minitest'
```

Comment:
Looks like this won't work for ruby 1.8. I would wrap this in:

```ruby
if RUBY_VERSION < '1.9'
  ...
end
```

And then wrap the test (or return early) similarly